### PR TITLE
test: init sharp integration test

### PR DESCRIPTION
**Pull request by ImBIOS**
### What does this PR do?

- #5852 

All 5 basic tests passing

- [ ] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [ ] Code changes

### How did you verify your code works?

I ran `bun test` locally

### CODE REVIEW

**Comment by paperclover** [comment]
thank you. sharp used to be a harder package to run because it depended on postinstall (see #7132 for status on that) but these should pass now.

File: test/integration/sharp/.gitignore
Comment (paperclover) [inline test/integration/sharp/.gitignore]: root of the entire project has a gitignore. please remove.

File: test/integration/sharp/README.md
Comment (paperclover) [inline test/integration/sharp/README.md]: this file can also be 💥

File: test/integration/sharp/index.ts
Comment (paperclover) [inline test/integration/sharp/index.ts]: 💥

File: test/integration/sharp/package.json
Comment (paperclover) [inline test/integration/sharp/package.json]: this `postinstall` should be removed. if sharp doesnt work out of box then it is a bug in bun.

File: test/integration/sharp/tsconfig.json
Comment (paperclover) [inline test/integration/sharp/tsconfig.json]: 💥

**Review by paperclover — CHANGES_REQUESTED** [review]
please cleanup the generated files from bun init. they are not needed. all that is needed is package.json, sharp.test.ts, bun.lockb, and bun.png

**Comment by ImBIOS** [comment]
@paperdave updated, and I changed `postinstall` of sharp became `trustedDependencies`, is that correct?

**Comment by paperclover** [comment]
Ideally this test should function without any configuration, no `trustedDependencies` set, but that might depend on the linked PR. I guess in the meantime it can be merged if it passes, and we will remove the trustedDependencies flag

**Comment by paperclover** [comment]
doesnt pass now but we should be able to revisit these tests after postinstall branch merges and it hopefully will work as-is.

File: test/integration/sharp/sharp.test.ts
Comment (dylan-conway) [inline test/integration/sharp/sharp.test.ts]:
```suggestion
    const inputBuffer = await sharp(path.join(import.meta.dir, "bun.png")).resize(200, 200).toBuffer();
```
also for the other uses of `bun.png`

**Comment by dylan-conway** [comment]
Thanks for these tests. I tested locally with the lifecycle script branch and they are passing

File: test/integration/sharp/package.json
Comment (dylan-conway) [inline test/integration/sharp/package.json]:
```suggestion
    "sharp": "0.33.0"
```

**Comment by ImBIOS** [comment]
@dylan-conway @paperdave updated

**Review by dylan-conway — APPROVED** [review]

**Comment by dylan-conway** [comment]
Thank you

**Pull request merged**
